pyaerocom/web/aerocom_evaluation.py

```diff
@@ -849,8 +854,17 @@ def find_coldata_files(self, model_name, obs_name, var_name=None):
                             var_name = self.model_config[model_name]['model_use_vars'][var_name]
                         except:
                             pass
+                        try:
+                            add_var_name = self.model_config[model_name]['model_add_vars'][var_name]
+                        except:
+                            add_var_name = None
                         if not m['var_name'] == var_name:
-                            match = False
+                            if isinstance(add_var_name,list):
+                                if not m['var_name'] in add_var_name:
+                                    match = False
```

Comment:
Could you add a test that reaches this code (i.e. handling of model_add_vars as list)?